mlflow/utils/credentials.py

```diff
@@ -41,3 +41,57 @@ def read_mlflow_creds() -> MlflowCreds:
         username=username_env or username_file,
         password=password_env or password_file,
     )
+
+
+def databricks_login(host, token=None, username=None, password=None):
+    """Log in Databricks and connect MLflow to Databricks workspace.
+
+    This method provides a simple way to connect MLflow to Databricks workspace. Users must specify
+    the host to databricks workspace, and set either `token` or `username`/`password` pair. This
+    method is useful if you want to use Databricks as the tracking server.
+
+    Args:
+        host: string, the URL of the Databricks workspace, must start with "https://", e.g.,
+            "https://community.cloud.databricks.com/".
+        token: string, the Databricks personal access token.
+        username: string, the username of the Databricks workspace, must be used with `password`.
+        password: string, the password of the Databricks workspace, must be used with `username`.
+
+    .. code-block:: python
+        :caption: Example
+
+        import mlflow
+
+        host = "https://community.cloud.databricks.com/"
+        token = "your-access-token"
+
+        mlflow.login(host, token)
+        with mlflow.start_run() as run:
+            mlflow.log_param("p", 0)
+    """
+    home = os.path.expanduser("~")
```

Comment:
How do we skip loging? The caller's environment may already has required credentials in `.databrickscfg`.

Reply:
Databricks CLI may have a function that checks whether you're logged in or not.